packages/slice-machine/src/features/customTypes/customTypesBuilder/UIDEditor.tsx

```diff
@@ -0,0 +1,97 @@
+import {
+  Box,
+  Button,
+  Dialog,
+  DialogActionButton,
+  DialogActions,
+  DialogCancelButton,
+  DialogContent,
+  DialogHeader,
+  Form,
+  FormInput,
+  Icon,
+  Text,
+} from "@prismicio/editor-ui";
+import { type KeyboardEvent, useState } from "react";
+
+import { getUidField, updateUidField } from "@/domain/customType";
+import { useCustomTypeState } from "@/features/customTypes/customTypesBuilder/CustomTypeProvider";
+
+export function UIDEditor() {
+  const [isOpen, setOpen] = useState(false);
+  const { customType, setCustomType } = useCustomTypeState();
+  const field = getUidField(customType);
+  const [label, setLabel] = useState(field?.config?.label ?? "");
```

Comment:
⚠️ #issue

You still are getting the label direction without using domain function here

Reply:
added domain function to get the label